index.js

```diff
@@ -12,6 +12,11 @@ function fetchJSON (url) {
     })
 }
 
+function coinList () {
+  const url = `${baseUrl}coinlist/`
```

Comment:
Should be:

    const url = `${baseUrl}all/coinlist`

Reply:
Right. I made this change but the tests are still failing.